lib/batch/institutionalize_psychiatry_and_behavorial_sciences.rb

```diff
@@ -0,0 +1,96 @@
+# Create parent collection 'Department of Psychiatry and Behavioral Sciences 
+# Collection' and institutionalize it
+# Create subcollections and add to parent collection:
+# - Department Events
+#   - Conferences
+#   - Commencement
+#   - Grand Rounds
+#   - Scholars Day
+# - Newsletters
+
+admin_group = 'Department-of-Psychiatry-and-Behavioral-Sciences-System-Admin'
+depositor = 'institutional-department-of-psychiatry-and-behavioral-sciences'
+
+dept_of_psych_and_behavioral_sci_sys_user = User.create!(
+  username: 'institutional-department-of-psychiatry-and-behavioral-sciences-system',
+  email: 'institutional-department-of-psychiatry-and-behavioral-sciences-system@northwestern.edu',
+  display_name: "Department of Psychiatry and Behavioral Sciences"
+)
+
+role = Role.create!(name: admin_group)
+
+#collection_user = User.find_or_create_via_username("rrr998")
+collection_user = User.find_or_create_via_username("gso1135")
+collection_user.add_role(role.name)
+collection_user.save!
+
+# The parent collection
+dept_of_psych_and_behavioral_sci_col = Collection.new(
+  title: 'Department of Psychiatry and Behavioral Sciences Collection',
+  tag: ["psychiatry", "psychology", "psych", "brain", "addiction",
+        "neuromodulation", "neuropsychology", "forensic psychiatry",
+        "consultation-liaison psychiatry", "geriatric psychiatry",
+        "behavioral health", "women's behavioral health",
+        "psychiatric education"],
```

Comment:
These tags can be set to a variable and re-used throughout.

Reply:
Okay, done.